### PR TITLE
Corrected the spelling of "test" -- cuz I'm a Dingus!

### PR DESCRIPTION
**Pull request by trevortomesh**
When I committed I wasn't sure how to spell the word "test" -- and I am not nearly drunk enough for this sort of thing.... so C'est la vie.

### CODE REVIEW

**Comment by holdingj** [comment]
This spelled gud.

**Comment by trevortomesh** [comment]
I think you did good isdd frits placeww wid ta spellingg! I sdon see a prablem.

**Comment by AdamHalawani** [comment]
http://www.talkenglish.com/grammar/grammar.aspx

File: README.md [hunk 1]
```diff
@@ -1,2 +1,2 @@
 # cs207winter2018
-This is the dtest repository for CS 207 Winter Term 2018
+This is the test repository for CS 207 Winter Term 2018
```
Comment (duncathan) [inline README.md:2]: You left out a period at the end of sentence; please fix before merging.

**Review by duncathan — CHANGES_REQUESTED** [review]

**Pull request merged**